chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -950,3 +950,8 @@ svap.edit.duplicate=Regulatory Text Citation and Approved Standard Version match
 apiKeyRequest.notFound=API Key Request not found.
 
 attestation.submissionPeriodException.cannotCreate=Cannot create Submission Period Exception for developer.
+
+form.formItem.required='%s' requires a response.
+form.formItem.cardinalitySingle='%s' may only have a single response.
+form.formItem.invalidResponse='%s' is not a valid response to the question '%s'.
+form.formItem.childQuestionNotApplicable=Responding to '%s' is not applicable based on the response to '%s'.
```

Comment:
Can there be a newline at the end of this file to avoid the checkstyle rule error about ending files with a newline?